moped-toolbox/amd_milestones_backfill/utils.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import sys
+
+import requests
+
+
+def get_logger(*, name, level=logging.INFO):
+    """Return a module logger that streams to stdout"""
+    logger = logging.getLogger(name)
+    handler = logging.StreamHandler(stream=sys.stdout)
+    formatter = logging.Formatter(fmt=" %(name)s.%(levelname)s: %(message)s")
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+    logger.setLevel(level)
+    return logger
+
+
+def make_hasura_request(*, query, variables, endpoint, admin_secret):
+    headers = {"X-Hasura-Admin-Secret": admin_secret}
+    payload = {"query": query, "variables": variables}
+    res = requests.post(endpoint, json=payload, headers=headers)
+    res.raise_for_status()
+    data = res.json()
+    try:
+        return data["data"]
+    except KeyError:
+        raise ValueError(data)
+
+
+# copied from atd-moped/moped-editor/src/utils/timelineTemplates.js
```

Comment:
It looks like this data also has a relationship to the database table `moped_milestones`. You might want to add an entry for this script in `moped-toolbox/README.md` and note what the script does and that a future user needs to make sure this data aligns to their database or they will get unexpected results. Not a major blocker by any means though.

Reply:
i totally missed that README. will do!